src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -118,6 +118,31 @@ public static Set<Skill> parseSkills(Collection<String> skills) throws ParseExce
         return skillSet;
     }
 
+
+    /**
+     * Parses a {@code String skill} into a {@code Skill} with importance.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code skill} is invalid.
+     */
+    public static Skill parseSkillImportant(String skill) throws ParseException {
```

Comment:
Would it be better to just use the regular `parseSkill` and set the `important` flag to `true`?

Reply:
Yeah did that for Unmarking but forgot to change it for Marking. Will change it once i get back